packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -85,6 +95,8 @@ function BasicColorList({
             pattern={pattern}
             isSelected={isSelected}
             isDisabled={isDisabled}
+            tabIndex={0 === i ? 0 : -1}
+            aria-label={__('Apply color', 'web-stories')}
```

Comment:
@barklund In the issue description it was mentioned that the labels need to be added to the new Color picker -- the only missing label I currently saw was for the Color Swatches. 

Were there any other labels you had in mind? 🤔 

It also seems to me that the focus styling is correct everywhere, but it is mentioned in the origin issue description. Let me know if I'm missing something.

Reply:
> Also, the color swatches should actually use named colors rather than plain hex descriptions as their labels (so, "pink", "salmon", etc), but that's a whole other can of worms!

I think we don't have to perhaps go that deep into this, considering there are also gradients available, probably not worth the effort as long as it's understandable generally. We could create a separate issue to look into that, including the Color Panels then.

Reading about `listbox`:
> Elements with the role listbox have an implicit aria-orientation value of vertical.

Also, it looks like arrow up/down should move to the next option, which would not be our case. Generally, it looks like `listbox` would be more for an actual list (assuming one option per row).

Perhaps `group` with `listitem` would be the best fit then? If we'd use `listitem` role, we'd ideally actually use the `ul` and `li` tags as well, right?

Thoughts?